docs/source/loading.mdx

```diff
@@ -80,6 +80,21 @@ You may have a 🤗 Datasets loading script locally on your computer. In this ca
 >>> dataset = load_dataset("path/to/local/loading_script", split="train")  # equivalent because the file has the same name as the directory
 ```
 
+### Edit loading script
+
+You can also edit a loading script from the Hub to add your own modifications. Download the dataset repository locally so any data files referenced by a relative path in the loading script can be loaded:
+
+```bash
+git clone https://huggingface.co/datasets/eli5
+```
+
+Make your edits to the loading script and then load it by passing its local path to [`~datasets.load_dataset`]:
+
+```py
+>>> from datasets import load_dataset
+>>> eli5 = load_dataset("/local/path/to/datasets/eli5")
```

Comment:
To be consistent with the paragraph above:
```suggestion
>>> eli5 = load_dataset("path/to/local/eli5")
```

Also note that there isn't necessarily a "datasets" directory once cloned from the Hub (and not from GitHub).